flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergStreamWriter.java

```diff
@@ -340,6 +364,10 @@ public void testPromotedFlinkDataType() throws Exception {
     SimpleDataUtil.assertTableRecords(location, expected);
   }
 
+
```

Comment:
please revert unnecessary empty line change

Reply:
Thank you, I have made the modification.